types/externs.d.ts

```diff
@@ -164,6 +164,8 @@ declare global {
       channel?: string
       /** Precomputed lantern estimates to use instead of observed analysis. */
       precomputedLanternData?: PrecomputedLanternData | null;
+      /** If true, lantern will use fixed values for RTT and server response time instead of observed network values. */
+      useFixedLanternRtt: boolean;
```

Comment:
feels weird to make this the only required property of this interface
```suggestion
      useFixedLanternRtt?: boolean;
```